swetify.css

```diff
@@ -234,11 +236,54 @@ text-shadow: 1px 1px 0 #3333ff;
          opacity:1;
          
       }
+	  
+	  
+	  .slider_div .center_div::-webkit-scrollbar{
+		display: none;
+	  }
+      
+	  .slider_div .slider-scrollbar{
+		height: 24px;
+		width: 100%;
+		display: flex;
+		align-items: center;
+	  }
 
-      .center_div::-webkit-scrollbar {
-        overflow: hidden;
-      }
-
+	
```

Comment:
Good work @Mrudula1205 ! I'd like to suggest a small change to make the carousel responsive on all devices. 
Currently, The left and right buttons are going out of the carousel many times and not always in the center.

To make it responsive try this -
Add `position: relative` to `slider-wrapper` 
Change `top: 50%` for `slide-button` by replacing `top: 235%`. 

Make this improvement and I will be happy to merge your PR.

Reply:
I have made the required changes